drm/DrmVirtgpu.h

```diff
@@ -37,10 +37,14 @@ extern "C" {
  * compatibility Keep fields aligned to their size
  */
 
+#define VIRTGPU_PARAM_RESOURCE_BLOB 3 /* DRM_VIRTGPU_RESOURCE_CREATE_BLOB */
```

Comment:
you should directly include kernel header file: include/uapi/drm/virtgpu_drm.h instead of define it here